static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -123,12 +125,13 @@ type IssueListProps = {
 };
 
 export function IssueList({issues, node, organization}: IssueListProps) {
+  const panelRef = useRef<HTMLDivElement>(null);
   if (!issues.length) {
     return null;
   }
 
   return (
-    <StyledPanel>
+    <StyledPanel ref={panelRef} width={panelRef.current?.clientWidth}>
```

Comment:
You need to ensure this doesnt conflict with the resizing behavior as we would now have two different min width definitions. I would use a container query here